src/geometries/Geometries.js

```diff
@@ -14,6 +14,7 @@ export * from './PolyhedronGeometry.js';
 export * from './RingGeometry.js';
 export * from './ShapeGeometry.js';
 export * from './SphereGeometry.js';
+export * from './SuperquadricGeometry.js';
```

Comment:
If we decide to add `SuperquadricGeometry`, I would prefer to add it as an addon so it should be placed in the `examples/jsm/geometries` directory.

Reply:
For better testing, it would be good to add it to one of the examples. How about removing the following block in `webgl_geometries`:

https://github.com/mrdoob/three.js/blob/2ff77e4b335e31c108aac839a07401664998c730/examples/webgl_geometries.html#L66C1-L68C25

and add a mesh with `SuperquadricGeometry` instead?

Reply:
In the meanwhile I suggest a different approach. Please host `SuperquadricGeometry` as a separate `npm` package and then create a new example that uses your library as an external dependency. We use this approach for other demos as well e.g. https://threejs.org/examples/?q=subdivision#webgl_modifier_subdivision

The import of the subvision library looks like so:
```js
"three-subdivide": "https://cdn.jsdelivr.net/npm/three-subdivide@1.1.2/build/index.module.js"
```
Your import could look like so:
```js
"three-superquadric": "https://cdn.jsdelivr.net/npm/three-superquadric@1.0.0/build/index.module.js"
``` 
Later in the code, the module is import like so:
```js
import { SuperquadricGeometry } from 'three-superquadric';
```